Exec/Make.PeleC

```diff
@@ -46,9 +46,15 @@ Bpack             += $(PP_SRC_HOME)/Make.package
 Blocs             += $(PP_SRC_HOME)
 
 # EOS
-EOS_HOME = $(PELE_PHYSICS_HOME)/Eos/$(strip $(Eos_dir))
-ifeq ($(Eos_dir),$(filter $(Eos_dir),Soave-Redlich-Kwong))
-   DEFINES += -DPELEC_USE_SRK
+EOS_HOME = $(PELE_PHYSICS_HOME)/Eos/
```

Comment:
Will we be able to just include the `Make.PelePhysics` here?

Reply:
Yes we should consolidate those... does it make sense in this PR? There's probably a bunch more from the file that should be consolidated.

Reply:
It can be done later if it's not trivial.